### ISSUE TITLE
Update peerDependencies to include React 17

### ISSUE THREAD

**Issue by just-boris** [react-grid-layout/react-resizable#151]
### Thanks for opening an issue!

Please select the type of issue you're reporting. For questions.

- [x] Bug
- [ ] Feature Request
- [ ] Question

### Problem Report

When using npm v7, the installation fails

```
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"0.14.x || 15.x || 16.x" from react-resizable@1.11.0
npm ERR! node_modules/react-resizable
npm ERR!   react-resizable@"1.11" from the root project
```

Latest NPM has a new more strict peer dependencies and it does not allow to install this package

#### System Info

Node Version: v15.6.0
Browser: n/a
OS: n/a

**Comment by zachsa** [comment]
Just out of interest... Is there any reason this can't be done simply by adding React 17 as a peer dependency?

**Comment by STRML** [comment]
It can easily be done, I would have expected someone to submit a PR by now.

**Comment by Lionprincipe** [comment]
hello, a way around is to use the flag --legacy-peer-dep 

  solution : npm install --legacy-peer-dep --save-exact react-resizable@1.11.0

source :https://stackoverflow.com/questions/64936044/fix-the-upstream-dependency-conflict-installing-npm-packages

**Comment by zachsa** [comment]
If it's as easy as just adding version 17 to the supported peer dependency list ... there's a pull request

**Comment by stromsky** [comment]
Could you publish new version to npm if this is fixed?

**Comment by KevinEverywhere** [comment]
Who is managing the library? If this is already fixed, why is it still unpublished? You never get a second chance to make a first impression, and React17 has been out for almost a year!

**Comment by STRML** [comment]
Hi. This is a free product. Who are we making an impression to?

**Comment by KevinEverywhere** [comment]
@STRML, I appreciate that price-point-zero is not a great incentive if one looks only for immediate financial remuneration. The feeling of doing a job well done and knowing that others are standing on the shoulders of the greats who preceded them is a motivation, and the impression is made for all of those who use software to achieve coding objectives. A workaround with the current build is possible by attaching --legacy-peer-deps to the installation. Thank you for what you have built, and it is useful and a generous contribution to the open source community. Thank you for your efforts.

npm install --save-exact react-resizable@1.11.0 @types/react-resizable@1.7.2 --legacy-peer-deps

The above has worked in a simple setting, and I hope that it will work as I build.

**Comment by just-boris** [comment]
@STRML as far as I see you are the only person with publish permissions for this package. So, all people have no option other than begging you for a release in this thread.

Since the the change is already merged, is there anything that it blocking the release? The change itself should be harmless, I already made a similar changes in a few other packages. This is the last one in our dependencies list, it would be great to see this resolved 😊

**Comment by STRML** [comment]
Released in `v1.11.1`.
